src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor

```diff
@@ -1,29 +1,108 @@
 ﻿@page "/build"
 @using ProductConstructionService.Client
 @using ProductConstructionService.Client.Models;
+@using TextCopy;
 @using ProductConstructionService.BarViz.Components;
 @inject IProductConstructionServiceApi PcsApi
+@inject IClipboard Clipboard
+@inject IJSRuntime JSRuntime
 
-<h3>Build</h3>
-@if (_build == null)
+@if (_build == null || _commit == null)
 {
     <p><em>Loading...</em></p>
 }
 else
 {
-    <h1>@_build.AzureDevOpsRepository</h1>
+    <div class="opposite">
```

Comment:
Do we need to define our own classes already? Can't we just go with what is available? The layout section of the help page has quite a few of these flex items